### ISSUE TITLE
fwa_bbox replacement?

### ISSUE THREAD

**Issue by sebdalgarno** [poissonconsulting/freshwateratlasbc#6]
currently there is no replacement fro fwa_bbox (e.g. finding bbox from collection/filter) in new API
would like to soft deprecate but no replacement and I think the function is useful

**Comment by sebdalgarno** [comment]
should this be an argument?, e.g. `fwa_stream_network("Sangan River", bbox = TRUE)
can also incorporate other transform functions like ST_Simplify, e.g. `fwa_stream_network("Sangan River", simplify = 100)`
or will this lead to argument bloat?

**Comment by smnorris** [comment]
@dr-jts `St_Envelope` is returned for each feature requested. V1 of FWA API uses `ST_Extent` https://github.com/tobinbradley/dirt-simple-postgis-http-api/blob/master/routes/bbox.js#L5.

**Comment by dr-jts** [comment]
> @dr-jts `St_Envelope` is returned for each feature requested. V1 of FWA API uses `ST_Extent`

Good point.  The `transform` solution doesn't provide the total extent of the query result.

It's inefficient to provide the bbox on every feature collection response (because the underlying query needs to be run twice).  So it seems like this should be a different query (perhaps called `bbox` as in the Dirt Simple API?).  This would be a non-standard extension to the OAPIF API.  I wonder if other OAPIF implementations support something like this?  Would be nice to follow an existing pattern.
